### PR TITLE
Support Teal with tl check

### PR DESCRIPTION
**Pull request by mundusnine**
https://github.com/teal-language/tl

### CODE REVIEW

**Comment by drmargarido** [comment]
Hello, thanks for the pull request.
Before merging the new linter I would just ask you to update the README with the new linter information.

**Comment by mundusnine** [comment]
@drmargarido  done.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Linters for https://github.com/rxi/lite
 * linter\_gocompiler - Uses the go vet command to display compilation errors in golang files
 * linter\_golint - Uses golint for golang files
 * linter\_luacheck - Uses luacheck for lua files
+* linter\_luacheck - Uses tl check for teal files
```
Comment (drmargarido) [inline README.md]: linter_teal instead of linter_luacheck

Comment (mundusnine) [reply README.md]: Sorry, updated.

**Pull request merged**